README.md

```diff
@@ -1,77 +1,172 @@
+- `-s` can be used to specify the status of pages you want to import. You can pass in a comma separated string of statuses or just a single status. The default is `publish,draft` if not specified.
 
-# Creating page models
+[View the import command source](wagtail_wordpress_import/management/commands/import_xml.py) to if you need to extend the command.
 
-There is a model mixin in the package which you should use in you own model, at least while running any imports.
+## Module documentation
 
-TODO
+- [Block Builder](docs/blockbuilder.md)
+- [Categories Import](docs/categories.md)
+- [Prefilters](docs/prefilters.md)
+- [Wordpress Shortcodes](docs/shortcodes.md)
+- [Yoast Import](docs/yoast.md)
 
-[View Documentation](docs/models.md)
+## Developer Tooling
 
-# Utility commands
+The XML file you are importing can be a large file. Some of the items in the XML file can be removed before you run the import script. This will reduce the time it takes to complete the import process.
 
-TODO 
+```bash
+python manage.py reduce_xml path/to/your/xmlfile.xml
+```
 
-## XML Tag Discovery
+ will remove all items of `<wp:comment>`
 
-`python manage.py extract_xml_mapping` [and add the path to your xml file]
+---
 
-[extract_xml_mapping.py](wagtail_wordpress_import/management/commands/extract_xml_mapping.py)
+To understand more about the type of HTML content formatting used in your Wordpress site we provide a reporting tool that will generate a table output in the console to show inline styles, html tags and shortcodes.
 
-This command will output a JSON file inside the json folder at the root of your project. The output can help you decide which fields should be included in your import as well as providing a nice short listing of all tags in the XLM for reference.
+```bash
+python manage.py analyze_html_content path/to/your/xmlfile.xml
+```
 
-## XML File Size Reduction
+---
 
-`python manage.py reduce_xml` [and add the path to your xml file]
+When testing imports you may need to remove all the imported pages and run it again. This script will run until all pages have been deleted and display progress in the console.
 
-[reduce_xml.py](wagtail_wordpress_import/management/commands/reduce_xml.py)
+```bash
+python manage.py delete_imported_pages [app] [page_model]
+# app and page_model are required arguments
+```
 
-This command will output a new xml file with `-reduced` appended to the xml file name and save it in the root `xml` folder.
+### Useful Shell Commands
 
-- Its default behaviour is to remove all comments and comment data.
-- It will also output some stats to explain the difference in lines in the xml file.
+Start a shell with `python manage.py shell`
+
+#### Delete all imported images
+
+```shell
+from wagtail.images.models import Image
++
+Image.objects.all().delete()
+```
+
+#### Delete all imported documents
+
+```shell
+from wagtail.documents.models import Document
++
+Document.objects.all().delete()
+```
```

Comment:
**Issue**: These commands both look like they will delete _all_ images and documents.